### ISSUE TITLE
[Bug] AR models not working inside the WKWebView

### ISSUE THREAD

**Issue by Mikilll94** [xamarin/Xamarin.Forms#9960]
### Description
I noticed that Augmented Reality models do not work inside the WKWebView. In Safari browser they are working fine.

In Apple docs there is gallery with AR models: https://developer.apple.com/augmented-reality/quick-look/

Here is the screenshot from Safari:

![4024AFF8-8E44-4547-97A2-895112D91C3E](https://user-images.githubusercontent.com/17653648/76687316-12971080-6623-11ea-9ec0-e1af81a67b8a.png)

And here is the screenshot from the WKWebView:

![37FC5DFA-3977-4EBC-9DC5-6ABE8B16D673](https://user-images.githubusercontent.com/17653648/76687328-280c3a80-6623-11ea-997b-1b9318a3a654.png)

In the Safari browser I can tap the AR model, the camera view opens and I can place the model inside the Augmented Reality.

In the WKWebView this is not working. I am tapping the AR model and nothing happens. Also as you can see from the screenshot the AR icon is not even showing up.

### Steps to Reproduce

1. Go to this website https://developer.apple.com/augmented-reality/quick-look/ in the Safari browser and in the WKWebView
2. As you can see, in the Safari browser you can tap the image to place it inside the Augmented Reality. In WKWebView this is not possible.

### Expected Behavior
AR models should work inside the WKWebView.

### Actual Behavior
AR models do not work inside the WKWebView.

### Basic Information

- Version with issue: 4.5.0.356
- IDE: Visual Studio For Mac 8.4.8
- Platform Target Frameworks: 
  - iOS:  Xamarin.iOS Version: 13.10.0.21

**Comment by jfversluis** [comment]
Do you have any evidence this should work in the `WKWebView`? I have tried searching for it, but came up with nothing. Also nothing that it's _not_ supported. But all the samples I see are using Safari or in-app. Not a WKWebView. It might not be supported by Apple in the first place?

**Comment by Mikilll94** [comment]
@jfversluis 
To be honest I don't know. I haven't also found info that this is not supported.

**Comment by Mikilll94** [comment]
@jfversluis 
Finally, I have found the evidence that this should work in the `WkWebView`.

From Apple docs:
https://developer.apple.com/documentation/arkit/previewing_a_model_with_ar_quick_look

"When the user clicks the link in Safari **or within a web view** that's displayed in your app, iOS presents your scene in an AR Quick Look view on your behalf. For more information, see Viewing Augmented Reality Assets in Safari for iOS."

**Comment by jfversluis** [comment]
I did see that mention, but I still don't see anything about the WKWebView specifically. Although I would agree that this should be the same thing. Just to try it I have created a iOS project without Forms with a WKWebView and tried this, also doesn't work. And to rule out absolutely everything I opened Xcode and made a pure native app that does this, also doesn't work. So either I'm missing something or this is a limitation/bug on the iOS side.

Xamarin.iOS: [Repro9960.zip](https://github.com/xamarin/Xamarin.Forms/files/4361770/Repro9960.zip)

Swift: 
[Repro9960_native.zip](https://github.com/xamarin/Xamarin.Forms/files/4361831/Repro9960_native.zip)
